src/Apps/Order/Components/__stories__/ItemReview.story.tsx

```diff
@@ -4,7 +4,7 @@ import { Flex } from "Styleguide/Elements/Flex"
 import { Section } from "Styleguide/Utils/Section"
 import { ItemReview } from "../ItemReview"
 
-storiesOf("Apps/Order/Components", module).add("ItemReview", () => {
+storiesOf("Apps/Order Page/Components", module).add("ItemReview", () => {
```

Comment:
Can you add a screenshot showing how this format appears in Storybooks?

Reply:
done